nautobot/wireless/views.py

```diff
@@ -57,6 +61,13 @@ def get_extra_context(self, request, instance=None):
         return context
 
 
+class SupportedDataRateCustomFieldsPanel(ObjectFieldsPanel):
+    def render_key(self, key, value, context):
+        if key == "mcs_index":
+            return format_html("<span>Modulation and Coding Scheme (MCS) Index</span>")
```

Comment:
Let's consider fixing this at the source by instead setting an appropriate `verbose_name` on the `mcs_index` field in `nautobot/wireless/models.py`, removing the need for this custom panel class?